src/services/infrastructure/entity-resolver/storage.space.resolver.service.ts

```diff
@@ -0,0 +1,183 @@
+import { Injectable } from '@nestjs/common';
+import { LogContext } from '@common/enums';
+import { Repository } from 'typeorm';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Hub } from '@domain/challenge/hub/hub.entity';
+import { Challenge } from '@domain/challenge/challenge/challenge.entity';
+import { StorageSpaceNotFoundException } from '@common/exceptions/storage.space.not.found.exception';
+import { InnovationPack } from '@library/innovation-pack/innovation.pack.entity';
+import { Platform } from '@platform/platfrom/platform.entity';
+import { Visual } from '@domain/common/visual/visual.entity';
+
+@Injectable()
+export class StorageSpaceResolverService {
+  constructor(
+    @InjectRepository(Hub)
```

Comment:
In a more generic service like this that needs to access multiple repositories and entities, EntityManager would be a better option. Otherwise, you end up with 10-15 imports just for the repositories (in time) and a 50-line constructor.